lib/open_project/scm/adapters/local_client.rb

```diff
@@ -28,6 +28,7 @@
 #++
 
 require 'open3'
+require 'find'
```

Comment:
Is this one really neccessary or was the fix to correctly qualify `::Find`?

Just asking because I had similar problems where the latter one was enough.